### PR TITLE
chore(docs): various typo fixes and doc updates

### PR DESCRIPTION
**Pull request by sanderson**
Various updates to the inline documentation to support the stdlib documentation that gets auto-generated. Changes include:

- Typo fixes
- Updated contributor spec
- Updated contrib packages to include contributors contact info
- Function introduced/deprecated version updates

### Done checklist
- [x] docs/SPEC.md updated
- [x] Test cases written **N/A**

### CODE REVIEW

File: stdlib/contrib/anaisdg/anomalydetection/mad.flux [hunk 1]
```diff
@@ -65,4 +70,4 @@ mad = (table=<-, threshold=3.0) => {
             )
 
     return output
-}
+}
```
Comment (Marwes) [inline stdlib/contrib/anaisdg/anomalydetection/mad.flux:73]: Nit: Missing newline

File: stdlib/contrib/bonitoo-io/servicenow/servicenow.flux [hunk 2]
```diff
@@ -220,4 +222,4 @@ endpoint = (url, username, password, source="Flux") =>
                                 ),
                         }
                     },
-                )
+                )
```
Comment (Marwes) [inline stdlib/contrib/bonitoo-io/servicenow/servicenow.flux:225]: Nit: Missing newline

**Review by Marwes — CHANGES_REQUESTED** [review]
Could you add back the newlines that are now missing from a bunch of the files? It pollutes the diff now and will pollute the diff again when someone else adds the newline back.

Otherwise LGTM.

File: stdlib/contrib/anaisdg/anomalydetection/mad.flux (hunk 1 above)
Comment (sanderson) [reply stdlib/contrib/anaisdg/anomalydetection/mad.flux:73]: Sorry about this. This appears to be applied by the version of the Flux formatter that's installed with the Flux VSCode extension.

**Review by Marwes — APPROVED** [review]

File: stdlib/contrib/anaisdg/anomalydetection/mad.flux (hunk 1 above)
Comment (Marwes) [reply stdlib/contrib/anaisdg/anomalydetection/mad.flux]: There seems to have been a discrepancy between the command line formatter and the LSP actually, https://github.com/influxdata/flux/pull/4905 should fix it.

**Review by lwandzura — APPROVED** [review]

**Pull request merged**